pyccl/tests/test_tracers.py

```diff
@@ -41,6 +41,38 @@ def get_tracer(tracer_type, cosmo=None, **tracer_kwargs):
     return tr, ntr
 
 
+def test_tracer_mag_0p4():
+    z = np.linspace(0., 1., 2000)
+    n = dndz(z)
+    # Small bias so the magnification contribution
+    # is significant (if there at all)
+    b = np.ones_like(z)*0.1
+    s_no = np.ones_like(z)*0.4
```

Comment:
I thought no magnification was s=0. Am I missing something?

Reply:
no magnification is s=2/5, I'm afraid :-)

Reply:
Ah, right. Might want to update the docstring here then: https://github.com/LSSTDESC/CCL/blob/master/pyccl/tracers.py#L94

Reply:
Hmm, but this is correct, right? If s is `None`, then it assumes s=0 (i.e. q=1) and returns the usual weak lensing kernel.

Reply:
You're right. I was confused by the `mag_bias` argument in `NumberCountsTracer`, where `mag_bias=None` effectively gives q=0, and `mag_bias` in `get_lensing_kernel`, where `mag_bias=None` gives q=1.